Frontend/src/Components/Faults/FaultsView.js

```diff
@@ -285,14 +285,217 @@ export default function Faults(props) {
     );
   };
 
+  const getHeartbeatString = () => {
+    let heartbeatStrings = [];
+
+    // Add any failing heartbeats
+    // The first element pushed should be capitalized, and any
+    // additional failing heartbeats should not be capitalized
+    if (_checkBooleanData("mainIO_heartbeat")) {
+      heartbeatStrings.push(heartbeatStrings.length === 0 ? "Main IO" : "main IO");
+    }
```

Comment:
It looks like this can only be the first element, so it should be capitalized by default